README.md

```diff
@@ -17,17 +17,17 @@ $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0
 
 I am a little bit confused about how to write in big O format. The following are just mathematical calculations with what I thought big O.
 
-$T(\log_{5}n)  \leq O(log_{2}n)$
+$T(\log_{5}n)  \leq c * O(log_{2}n)$ ---1st equation
 
-$T(\log_{5}n)  \leq c * O(log_{2}n)$ 
+$\frac{T(\log_{5}n)}{O(log_{2}n)} \leq c$
 
-$if\ c = log_{5}2\$
+$When \frac{T(\log_{5}n)}{O(log_{2}n)} = c$
 
-$T(\log_{5}n) \leq O(log_{5}2$ * $\log_{2}n)$ 
+$c = \frac{lg_{n}}{lg_{5}} * \frac{lg_{2}}{lg_{n}} = \frac{lg_{2}}{lg_{5}} = \log_{5}2$
```

Comment:
You can't assume a specific $c$. You need to show the general case for all elements of both $O$ sets.